db/post_migrate/20230119024157_remove_topic_count_from_tags.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "migration/column_dropper"
+
+class RemoveTopicCountFromTags < ActiveRecord::Migration[7.0]
+  DROPPED_COLUMNS ||= { tags: %i[topic_count] }
+
+  def up
+    DROPPED_COLUMNS.each { |table, columns| Migration::ColumnDropper.execute_drop(table, columns) }
```

Comment:
Usually we like to leave columns around but this column and its results can easily be rebuilt so there is no need for us to play it safe here.

Reply:
Yes I agree, it's not really worthwhile information that can't be recreated, thanks for adding the note :+1: